### PR TITLE
docs: update auth server docs

### PR DESCRIPTION
**Pull request by clouserw**
Because:

* Our docs are confusingly in two places and some are outdated

This commit:

* Moves docs to the ecosystem docs and removes outdated docs

### CODE REVIEW

File: packages/fxa-auth-server/docs/device_registration.md
Comment (clouserw) [inline packages/fxa-auth-server/docs/device_registration.md:1]: https://mozilla.github.io/ecosystem-platform/relying-parties/how-tos/device-registration

File: packages/fxa-auth-server/docs/metrics-events.md
Comment (clouserw) [inline packages/fxa-auth-server/docs/metrics-events.md:1]: Metrics stuff is being replaced now so we'll update the metrics docs as that is written.

File: packages/fxa-auth-server/docs/oauth/api.md
Comment (clouserw) [inline packages/fxa-auth-server/docs/oauth/api.md:1]: https://mozilla.github.io/ecosystem-platform/api

File: packages/fxa-auth-server/docs/oauth/clients.md
Comment (clouserw) [inline packages/fxa-auth-server/docs/oauth/clients.md:1]: https://mozilla.github.io/ecosystem-platform/reference/oauth-details#configuring-oauth-clients

File: packages/fxa-auth-server/docs/oauth/prompt-none.md
Comment (clouserw) [inline packages/fxa-auth-server/docs/oauth/prompt-none.md:1]: https://mozilla.github.io/ecosystem-platform/reference/oauth-details#promptnone-support

File: packages/fxa-auth-server/docs/oauth/scopes.md
Comment (clouserw) [inline packages/fxa-auth-server/docs/oauth/scopes.md:1]: https://mozilla.github.io/ecosystem-platform/reference/oauth-details#oauth-scopes

File: packages/fxa-auth-server/docs/overview.md
Comment (clouserw) [inline packages/fxa-auth-server/docs/overview.md:1]: Moved relevant parts to auth-server readme

File: packages/fxa-auth-server/docs/recovery_keys.md
Comment (clouserw) [inline packages/fxa-auth-server/docs/recovery_keys.md:1]: https://mozilla.github.io/ecosystem-platform/reference/tokens#account-recovery-tokens

File: packages/fxa-auth-server/docs/schema.md
Comment (clouserw) [inline packages/fxa-auth-server/docs/schema.md:1]: https://mozilla.github.io/ecosystem-platform/reference/database-structure

File: packages/fxa-auth-server/docs/service_notifications.md
Comment (clouserw) [inline packages/fxa-auth-server/docs/service_notifications.md:1]: We shouldn't encourage anyone else to use this.  Use event broker instead.

File: packages/fxa-auth-server/README.md [hunk 1]
```diff
@@ -1,148 +1,26 @@
...
-To forward Stripe webhooks to your local, you can use the [Stripe CLI](https://stripe.com/docs/stripe-cli).
-You'll need to `stripe login` to authenticate before you can start forwarding.
-
-To start webhook forwarding, run:
-
-    stripe listen --forward-to localhost:9000/v1/oauth/subscriptions/stripe/event
-
-The first time you run this, you'll need to grab the Stripe webhook secret displayed there, and add that to your secrets.json as described in [secrets](#Secrets).
-
-## Secrets
```
Comment (clouserw) [inline packages/fxa-auth-server/README.md:59]: Moved to https://mozilla.github.io/ecosystem-platform/tutorials/development-setup#secrets

File: packages/fxa-auth-server/README.md [hunk 2]
```diff
@@ -1,148 +1,26 @@
...
-- `test/mail_helper.js` on port 9001
-- `./node_modules/fxa-customs-server/bin/customs_server.js` on port 7000
-
-When you `Ctrl-c` your server, all 4 processes will be stopped.
-
-To start the server in dev MySQL store mode (ie. `NODE_ENV=dev`), run:
-
-    npm run start-mysql
-
-To forward Stripe webhooks to your local, you can use the [Stripe CLI](https://stripe.com/docs/stripe-cli).
```
Comment (clouserw) [inline packages/fxa-auth-server/README.md:50]: Moved to https://mozilla.github.io/ecosystem-platform/tutorials/development-setup#stripe

File: packages/fxa-auth-server/CONTRIBUTING.md
Comment (clouserw) [inline packages/fxa-auth-server/CONTRIBUTING.md:1]: Should be using the top level CONTRIBUTING

File: packages/fxa-auth-server/docs/endpoint_data.md [hunk 3]
```diff
@@ -1,416 +0,0 @@
-## /account/create
```
Comment (vbudhram) [inline packages/fxa-auth-server/docs/endpoint_data.md:1]: 👍🏽

File: packages/fxa-auth-server/docs/metrics-events.md
Comment (vbudhram) [reply packages/fxa-auth-server/docs/metrics-events.md:1]: It appears that some of this data isn't in the ecosystem docs. Do you think we should keep it around until we do the update to metrics?

**Review by vbudhram — APPROVED** [review]
@clouserw Thank you!

**Pull request merged**